manticore/core/smtlib/solver.py

```diff
@@ -588,62 +620,107 @@ def _optimize_fancy(self, constraints: ConstraintSet, x: BitVec, goal: str, max_
             raise SolverError("Optimize failed")
 
     def get_value(self, constraints: ConstraintSet, *expressions):
+        values = self.get_value_in_batch(constraints, expressions)
+        if len(expressions) == 1:
+            return values[0]
+        else:
+            return values
+
+    def get_value_in_batch(self, constraints: ConstraintSet, expressions):
         """
         Ask the solver for one possible result of given expressions using
         given set of constraints.
         """
-        values = []
+        # print(''.join(traceback.format_stack()))
+        # print()
```

Comment:
Let's remove these, along with any other commented print statements that we're unlikely to need.

Reply:
Done.